Library/Homebrew/cask/lib/hbc/cli/internal_dump.rb

```diff
@@ -7,7 +7,7 @@ def initialize(*)
       end
 
       def run
-        raise CaskError, "Dump incomplete." if dump_casks == :incomplet
+        raise CaskError, "Dump incomplete." if dump_casks == :incomplete
```

Comment:
CC @reitermarkus to check this is correct.

Reply:
Yes.